extra/exports

```diff
@@ -182,6 +182,8 @@ fiber_set_cancellable
 fiber_set_joinable
 fiber_sleep
 fiber_start
+fiber_set_arg
+fiber_get_arg
```

Comment:
Please, keep the content of the file sorted. See a comment in the beginning of the file to find how to sort it.

Reply:
I did `sort -f`, and this resulted in some of the already commited lines to swap but I decided no to commit those ones.

Reply:
Indeed. It means some other patch slipped through review somehow.